annif/openapi/annif.yaml

```diff
@@ -134,6 +134,41 @@ paths:
               schema:
                 $ref: '#/components/schemas/Problem'
       x-codegen-request-body-name: documents
+  /detect-language:
+    post:
+      tags:
+      - Languages detection
+      summary: detect the language of a text given a list of candidate languages
+      operationId: annif.rest.detect_language
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+              - text
+              - candidates
+              properties:
+                text:
+                  type: string
+                  description: input text
+                  example: A quick brown fox jumped over the lazy dog.
+                candidates:
+                  type: array
+                  description: candidate languages as ISO 639-1 codes
+                  items:
+                    type: string
+                    maxLength: 2
```

Comment:
Should be 3, as some valid BCP 47 language tags (e.g. `enm`, `hbs`) may have 3 characters